### ISSUE TITLE
Training slowly

### ISSUE THREAD

**Issue by wang5566** [marvis/pytorch-mobilenet#18]
I trained the imagenet dataset with 4 gpu 1080ti but it took about 17 hours for 3 epochs and accuracy got 35% top1 , so I want to know how long did it take when you trained your model with your accuracy?
